src/System.Data.SqlClient/System.Data.SqlClient.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```

Comment:
Can this file be excluded from the change ?

Reply:
I've tried. Visual studio really _really_ wants me to upgrade the solution though and doesn't like me trying not to. Unless it's harmful it might be a good idea just to take the update at some point.